app/assets/stylesheets/components/_contact_panels.scss

```diff
@@ -0,0 +1,90 @@
+.l-contact-panel {
+  @include column(4);
+}
+
+.contact-panel {
+  @include column(12);
+
+  color: $color-grey-medium-dark;
+  text-align: center;
+  padding: $baseline-unit*2 0;
+
+  display: flex;
+  flex-direction: column;
+  align-items: center;
```

Comment:
Could we make use of the flex-box implementation here to align the child elements of each panel so that the "Launch chat" buttons align in a similar way to frontend?

Reply:
These buttons are aligned